sites/gc-page-feedback/includes/strings.liquid

```diff
@@ -0,0 +1,58 @@
+{%- comment -%}
+Set string variables
+{%- endcomment -%}
+
+{%- assign opuft_yes_en = "Yes" -%}
+{%- assign opuft_no_en = "No" -%}
+{%- assign opuft_problem1_label_en = "I can’t <strong>find</strong> the information" -%}
+{%- assign opuft_problem2_label_en = "The information is hard to <strong>understand</strong>" -%}
+{%- assign opuft_problem3_label_en = "There was an error or something <strong>didn’t work</strong>" -%}
+{%- assign opuft_problem4_label_en = "Other reason" -%}
+
+{%- assign opuft_yes_fr = "Oui" -%}
+{%- assign opuft_no_fr = "Non" -%}
+{%- assign opuft_problem1_label_fr = "Je ne peux pas <strong>trouver</strong> l’information" -%}
+{%- assign opuft_problem2_label_fr = "L’information est difficile à <strong>comprendre</strong>" -%}
+{%- assign opuft_problem3_label_fr = "Il y avait une erreur ou quelque chose <strong>ne fonctionnait pas</strong>" -%}
+{%- assign opuft_problem4_label_fr = "Autre raison" -%}
+
+{%- assign opuft_yes_value = opuft_yes_en | append: "-" | append: opuft_yes_fr -%}
+{%- assign opuft_no_value = opuft_no_en | strip_html | append: "-" | append: opuft_no_fr | strip_html -%}
+{%- assign opuft_problem1_value = opuft_problem1_label_en | strip_html | append: "-" | append: opuft_problem1_label_fr | strip_html -%}
+{%- assign opuft_problem2_value = opuft_problem2_label_en | strip_html | append: "-" | append: opuft_problem2_label_fr | strip_html -%}
+{%- assign opuft_problem3_value = opuft_problem3_label_en | strip_html | append: "-" | append: opuft_problem3_label_fr | strip_html -%}
+{%- assign opuft_problem4_value = opuft_problem4_label_en | strip_html | append: "-" | append: opuft_problem4_label_fr | strip_html -%}
+
+{%- if page.language == "en" -%}
```

Comment:
@duboisp Is the property is `page.lang` or `page.language`. Because I'm seeing both throughout the codebase.
We should have `page.lang` for the 2-character version and `page.language` for the longer version with the country code (i.e "fr-CA")

Reply:
> @duboisp Is the property ``page.lang`` or ``page.language``. I'm seeing both throughout the codebase.

My guess is that the "preferred" variable name is probably ``page.lang``.

Having said that, it looks like [GCWeb technically supports both ``page.lang`` and ``page.language``](https://github.com/wet-boew/GCWeb/blob/2ceca4dc0e4fce4afc567f10968b7b5a668eb687/sites/includes/variable-core.liquid#L12-L21). But ``page.lang`` is checked first. My guess is that ``page.language`` is probably a carry-over from the older handlebars incarnation of GCWeb (since WET uses ``language``).

Some page variable names changed when GCWeb transitioned from handlebars to Jekyll. But I don't think any attempt was made to update outdated ones on GCWeb's pre-existing pages. That's why variable names are all over the place atm. FWIW I fixed some totally-broken ones in #1945 but didn't bother with ones that worked.

> We should have page.lang for the 2-character version and page.language for the longer version with the country code (i.e "fr-CA")

I don't think there'd be any value in going down that path.

Both WET + GCWeb are already able to cope with long language codes as a result of wet-boew/wet-boew#9272. More info can be found in [WET's core documentation page](https://wet-boew.github.io/v4.0-ci/unmin/docs/ref/core/core-en.html).